src/components/TabbedCard/TabbedCard.react.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+
+import * as React from "react";
+import Tab from "./Tab.react";
+import Card from "../Card/Card.react";
+
+type Props = {|
+  +initialTab: string,
+  +children: React.ChildrenArray<React.Element<typeof Tab>>,
+|};
+
+type State = {|
+  selectedTitle: string,
+|};
+
+class TabbedCard extends React.PureComponent<Props, State> {
+  state = {
+    selectedTitle: this.props.initialTab,
+  };
+
+  render(): React.Node {
+    const tabs = React.Children.toArray(this.props.children);
+    return (
+      <Card>
+        <Card.Header>
+          <ul className="nav nav-tabs TabbedCard_header_tabs">

```

Comment:
Could this be a new `Tabs` component that is built upon our existing `Nav` components?